### ISSUE TITLE
Not able to add a new device that is connected to SiLA service

### ISSUE THREAD

**Issue by crossj-c** [ebastidas/sila#3]
Why I am not able to add a new device that is connected to SiLA service.  It keeps report "soap error".

**Comment by ebastidas** [comment]
Hi, 
Could you be more specific in your environment. Are you trying to connect your SiLA device to our demo website or to your local SOFIA installation? Did you try to test your soap service (reset or other SiLA commands) via another soap consumer (not necessarily SiLA compliant, only a soap test tool)?

**Comment by crossj-c** [comment]
Hi ebastidas, thank you for support. I managed to get rid of the problem.
It was a strange connection problem, Sofia keeps reporting ECONNREFUSED error while creating a soap client.  After I filed the reported, I found that Sofia try to create a client against 127.0.0.1 (localhost) though I specified a proper uri to a remote PC (sila provider).  I have no idea why sofia doesn't talk to remote uri instead try to making connect to localhost.  After I changed network settings in sila provider, the connection problem is gone.  I can't explain the resolution either.  Since I was busy on others, I didn't invest too much time on this issue anymore.

**Comment by ebastidas** [comment]
Yes, the default settings make SOFIA run only locally (both the SiLA consumer and using only local SiLA devices). To make it work in a network you only need to run SOFIA with the following command  (as specified in the README file): 
`env SOFIA_IP=192.168.0.1 SOFIA_PORT=80 node run.js`
Where `192.168.0.1` is the IP of the machine that's hosting SOFIA, then you can connect to other SiLA devices in your network.
